draft-ietf-rats-architecture.md

```diff
@@ -330,14 +332,15 @@ implicitly trusted is often referred to as a Root of Trust.
 It is important to prevent replay attacks where an attacker replays
 old Evidence or an old Attestation Result that is no longer correct.
 To do so, some mechanism of ensuring that the Evidence and Attestation
-Result are fresh, meaning that there is some degree of assurance
+Results are fresh, meaning that there is some degree of assurance
 that they still reflect the latest state of the Attester, and that any Attestation
-Result was generated using the latest Appraisal Policy for Evidence.
+Result was generated using the latest Appraisal Policy for Evidence -- and
+latest Evidence, accordingly.
 There is, however, always a race condition possible in that the
 state of the Attester, and the Appraisal Policy for Evidence, may
-change immediately after the Evidence or Attestation Result was generated.
+change immediately after the Evidence or Attestation Results were generated.
 The goal is merely to narrow the time window to something the Verifier
-(for Evidence) or Relying Party (for an Attestation Result) is willing to accept.
+(for Evidence) or Relying Party (for an Attestation Result) is willing to accept. Effectively, the burden of appraisal with respect to freshness is also a duty of the Verifier.
```

Comment:
This part doesn't make sense for Attestation Results in my opinion.

Reply:
Correct. That is a content oversight! It's on my list, but I made an error here by omitting that.

Attestation Results have a certain "time interval of validity", which we have not talked about at all, yet? I assume?